routemaster/subscription.rb

```diff
@@ -0,0 +1,22 @@
+module Routemaster
+  class Subscription
```

Comment:
I know this wasn't done elsewhere but I'd suggest namespacing this under `Routemaster::Client`.
Rationale: the client and drain are loaded in the server's codebase for testing, and clashes are gonna be a nightmare :)